sos/plugins/kdump.py

```diff
@@ -28,21 +29,34 @@ class RedHatKDump(KDump, RedHatPlugin):
     packages = ('kexec-tools',)
 
     def setup(self):
+        initramfs_img = "/boot/initramfs-" + platform.release() \
+                        + "kdump.img"
```

Comment:
This sets on a Fedora30 something like: `/boot/initramfs-5.1.20-300.fc30.x86_64kdump.img` while I see files like:

    # ls /boot/initramfs-*
    /boot/initramfs-0-rescue-98da600420ce46698fbf59f8bd80371b.img
    /boot/initramfs-5.1.16-300.fc30.x86_64+debug.img
    /boot/initramfs-5.1.16-300.fc30.x86_64.img
    /boot/initramfs-5.1.19-300.fc30.x86_64+debug.img
    /boot/initramfs-5.1.19-300.fc30.x86_64.img
    /boot/initramfs-5.1.20-300.fc30.x86_64+debug.img
    /boot/initramfs-5.1.20-300.fc30.x86_64.img
    #

(on a RHEL7.6, there is such a file with `initramfs_img` filename).

Reply:
Kdump service generates boot/initramfs-`uname -r`kdump.img.

For example,

Fedora 28:
initramfs-4.20.11-100.fc28.x86_64.img
initramfs-4.20.11-100.fc28.x86_64kdump.img

RHEL 7.7:
initramfs-3.10.0-1040.el7.ppc64le.img
initramfs-3.10.0-1040.el7.ppc64lekdump.img

-Sourabh Jain

Reply:
Well I listed `/boot/initramfs-*` from a Fedora system I am using, and I do not see any `*kdump.img` file :) But if that does not matter for the sake of troubleshooting that requires only `*kdump.img` files, I am OK with the patch.